app/controllers/games_controller.rb

```diff
@@ -32,3 +40,8 @@ def game_params
     params.require(:game).permit(:name, :number_of_moves, :black_player_id, :white_player_id, :game_status)
   end
 end
+
+  def black_and_white_player?
```

Comment:
Yeah, I was about to add a different functionality about not letting players move pieces on the board before the opponent entered the game but forgot to get rid of it.